src/web/client/extension.ts

```diff
@@ -459,3 +462,19 @@ function isActiveDocument(fileFsPath: string): boolean {
         WebExtensionContext.fileDataMap.getFileMap.has(fileFsPath)
     );
 }
+
+async function logArtemisTelemetry() {
+    const orgId =  WebExtensionContext.urlParametersMap.get(
```

Comment:
add a try catch on this call - we dont want the extension to fail loading in case this function fails.